### PR TITLE
Allow larger build limit for projects

### PR DESCRIPTION
**Pull request by ewpatton**
According to [the App Engine documentation](https://cloud.google.com/appengine/docs/standard/java/issue-requests#using_standard_runtime_network_classes), in the Java 8 runtime the limit on HTTP requests is no longer enforced if the standard Java network connection APIs are used. This change makes it so that the max size of an app is configured via appengine-web.xml. In my testing on App Engine, the ReceiveBuildServlet would throw OOM errors using F1 instances on larger apps, so I've set the default to 30 MB for now (up from the 10 MB hard limit using URLFetch).

Change-Id: Ief64d60a120d1ca9e624e3bc9dd57224064585b0

### CODE REVIEW

**Comment by jisqyv** [comment]
@ewpatton Note: in production on ai2, we are using F2 instances

**Comment by ewpatton** [comment]
@jisqyv Yes. When I tested with F2 instances it worked with a 50 MB limit. I used 30 MB conservatively.

File: appinventor/appengine/src/com/google/appinventor/server/project/youngandroid/YoungAndroidProjectService.java [hunk 1]
```diff
@@ -697,16 +703,16 @@ public RpcResult build(User user, long projectId, String nonce, String target,
...
-        int zipFileLength = zipFile.getContent().length;
-        String lengthMbs = format((zipFileLength * 1.0)/(1024*1024));
+      if (zipFile.getContent().length > MAX_PROJECT_SIZE.get() * MB) {
+        double size = (double) zipFile.getContent().length / MB;
         RuntimeException exception = new RuntimeException(
-            "Sorry, can't package projects larger than 10Mb."
-            + " Yours is " + lengthMbs + "MB.");
+            String.format(Locale.getDefault(), ERROR_LARGE_PROJECT, MAX_PROJECT_SIZE.get(), size)
+        );
         CrashReport.createAndLogError(LOG, null,
```
Comment (jisqyv) [inline appinventor/appengine/src/com/google/appinventor/server/project/youngandroid/YoungAndroidProjectService.java:711]: Perhaps we should get rid of the call to CrashReport, as this isn't really a crash. Calling crash report results in a SEVERE error being recorded in the log along with the stack trace.

Look at line 774. Probably want to get rid of the reference to larger then 5Mb (in the IOException handler).

Comment (ewpatton) [reply appinventor/appengine/src/com/google/appinventor/server/project/youngandroid/YoungAndroidProjectService.java]: Done

**Pull request merged**